pages/guides/developing/encrypted-vod.en-US.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: 'Upload encrypted files to Livepeer'
+description: 'Learn how to upload and playback an encrypted video asset'
+---
+
+import { Callout } from 'nextra-theme-docs';
+
+# Encrypted VOD with Livepeer
+
+Livepeer enables you to upload encrypted video content or import it from decentralized storages. It creates a playback URL that can be covered by playback policies and access control.
+
+<Callout>
+	This guide utilize snippets in Golang as examples.
+</Callout>
```

Comment:
Let's change these to JavaScript before merging, this feature will be used from the browser so JS is our main language. You probably already have that code from that dev-server.com sample app right? Could substitute here. 

Can still keep a version in Go, but I think it won't help many Go devs if this is the only guide in our docs with Go samples, so maybe just keep it consistent the other docs with only JS.